src/Avatar/stringToColor.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Generate a background color for avatars from the given string
```

Comment:
Is there a limitation as being a "background"? like it's always readable if you set a black text on it? Just wondering. If not maybe you can generalise the comment.